src/Components/list-of-reviews.jsx

```diff
@@ -0,0 +1,38 @@
+import { useState, useEffect } from 'react'
+
+
+function CreateListOfReviews() {
+
+const [listOfReviews, setListOfReviews] = useState([])
+
+
+useEffect(()=>{
+
+    fetch("https://sidneys-games-ii.onrender.com/api/reviews")
```

Comment:
Although not crucial, you may want to extract the fetching behaviour here into a `utils` or `api` file - you'll have a lot of functions like this before the week is out!